v3.0/getting-started/kubernetes/installation/hosted/kubernetes-datastore/policy-only/1.7/calico.yaml

```diff
@@ -201,15 +201,15 @@ apiVersion: apiextensions.k8s.io/v1beta1
 description: Calico Felix Configuration
 kind: CustomResourceDefinition
 metadata:
-   name: felixconfigs.crd.projectcalico.org
+   name: felixconfigurations.crd.projectcalico.org
 spec:
   scope: Cluster
   group: crd.projectcalico.org
   version: v1
   names:
-    kind: FelixConfig
-    plural: felixconfigs
-    singular: felixconfig
+    kind: FelixConfiguration
```

Comment:
@bcreane @caseydavenport not sure if we change the manifest for a release version without doing a minor release. Should I keep this or wait until the next patch release?

Reply:
We should update v3.0 and master